docs/tutorials/remote_soc.rst

```diff
@@ -0,0 +1,220 @@
+
+See the :ref:`Installation <Installation>` section for information on how to install SiliconCompiler, and the :ref:`Remote Processing <Remote Processing>` section for instructions on setting up the remote workflow.
+
+[TODO / CR feedback: Add a screenshot here (or at the bottom)?]
+
+Download PicoRV32 Verilog Code
+------------------------------
+
+The heart of any digital design is its HDL code, typically written in a language such as Verilog or VHDL. High-level synthesis languages are gaining in popularity, but most of them still output their final design sources in a traditional HDL such as Verilog.
+
+PicoRV32 is an open-source implementation of a small RISC-V CPU core, the sort you might find in a low-power microcontroller. Its source code, license, and various tooling can be found `in its GitHub repository <https://github.com/YosysHQ/picorv32>`_.
+
+The repository contains many files, but the core CPU design is located in a single file called ``picorv32.v`` at the root of the repository.
+
+Create a new directory for this project, and copy the ``picorv32.v`` file into it.
+
+Build the PicoRV32 Core using SiliconCompiler
+---------------------------------------------
+
+Before we add the complexity of a RAM macro block, let's build the core design using the open-source Skywater130 PDK. Copy the following build script into the same directory which you copied ``picorv32.v`` into::
+
+    import siliconcompiler
+
+    chip = siliconcompiler.Chip('picorv32')
+    chip.load_target('skywater130_demo')
+    chip.input('picorv32.v')
+    chip.set('option', 'remote', True)
+    chip.run()
+
+If you run that example as a Python script, it should take approximately 10-15 minutes to run if the servers are not too busy. We have not added a RAM macro yet, but this script will build the CPU core with I/O signals placed pseudo-randomly around the edges of the die area. Once the job finishes, you should receive a screenshot of your final design, and a report containing metrics related to the build in the ``build/picorv32/job0/`` directory.
+
+For the full GDS-II results and intermediate build artifacts, you can install the EDA tools on your local system, and run the same Python build script with the :keypath:`option, remote` parameter set to ``False``. We are not returning the full results during this early beta period because we want to minimize bandwidth, and we believe that our public beta is currently best suited for rapid prototyping and design exploration.
```

Comment:
We should make the "We are not returning the full results during this early beta period because we want to minimize bandwidth, and we believe that our public beta is currently best suited for rapid prototyping and design exploration." a .. note:: so people see it clearly (also add that to the remote processing).